sst_wrapper/envs/bc_gym_wrapper.py

```diff
@@ -33,7 +43,7 @@ def set_goal_position(self):
         """
         Sets the goal position of the robot. The states that do not mark the initial position of the robot is set to be 0.1.
         """
-        goal_pose = self.env._reward_provider._state.current_goal_pose()
+        goal_pose = self.env._reward_provider._state.path[-1]  #pylint: disable=protected-access
```

Comment:
why this change

Reply:
This is more general, and less likely to have problems with shining.

Reply:
naaah let's do it the proper way